students/alex_whitty/lesson3/mailroom.py

```diff
@@ -0,0 +1,106 @@
+import sys  # imports go at the top of the file
+
+
+# Data Structure
+donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "Elon Musk": [2263.23, 3300.87, 15432.0],
+            }
+
+
+
+# Input/Output
+prompt = "\n".join(("Thanking your donors for their generous gifts!",
+          "Please choose from below options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Exit",
+          ">>> "))
+
+# Features
+def thank_you():
+
+    answer = input("Please enter Full Name.>>>")
+    if answer.lower() == 'list':
+        for donor in donor_db.keys():
+            print(donor)
+    else:
+        if answer in donor_db:
+            current_donor(answer)
+        else:
+            add_donor()
```

Comment:
This one should be _add_donor(answer)_, so that the name of the donor is passed to _add_donor()_, preventing the user from being asked for the name again (a bit confusing).